src/plone/exportimport/importers/content.py

```diff
@@ -19,6 +20,7 @@ class ContentImporter(BaseImporter):
     name: str = "content"
     metadata: types.ExportImportMetadata = None
     languages: types.PortalLanguages = None
+    dropped: set = set()
```

Comment:
You can't do this safely. The set instance will be shared between all instances of ContentImporter. Best to initialize it in __init__

Reply:
@davisagli Fixed.